blank_project/src/config.js

```diff
@@ -18,6 +18,8 @@
                     networkId: 'local',
                     nodeUrl: 'http://localhost:3030',
                     keyPath: '~/.near/validator_key.json',
+                    walletUrl: 'http://localhost:4000',
```

Comment:
I suggest moving `_wallet` to something like `_wallet/login`, then whole `walletUrlSuffix` thing won't be needed.

Reply:
It needs to be /wallet/login/?title=...

Reply:
> It needs to be /wallet/login/?title=...

so yes, you can put `login` file into `wallet` folder and you'll get this.  Anything I'm missing?

Reply:
If you do that and run npm run start, http://localhost:4000/wallet/login/?(needed url params)  doesn't work

Reply:
updated to use /wallet/login/index.html which works with the current url template